decentraland/unity-renderer
# fix: duplicated sprites in UI atlases

## What does this PR change?

`Fixes #4209 `

Moved all the UI textures to `Common` folder.
Every subfolder in the UI have its own atlas containing its own sprites.

## How to test the changes?

1. Go to: https://play.decentraland.org/?explorer-branch=fix/duplicated-sprites-in-atlases
2. Navigate through all the UI
3. No missing icons in the UI should be missing

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
